examples/monitor-with-externalConfigMap/grafana/tidb-monitor.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
Please add a readme file about what's this for and how to use the files.

Reply:
ok.